mxnet/training/docker/1.6.0/py3/Dockerfile.gpu

```diff
@@ -151,16 +151,18 @@ RUN ${PIP} install --no-cache --upgrade \
     scikit-learn==0.20.4 \
     scipy==1.2.2 \
     dgl-cu101==0.4.1 \
-    git+git://github.com/dmlc/gluon-nlp.git@v0.9.0 \
-    urllib3==1.24 \
+    gluonnlp==0.9.1 \
+	gluoncv==0.6.0 \
+	urllib3==1.25.8 \
     python-dateutil==2.8.0 \
     tqdm==4.39.0 \
-    sagemaker-experiments==0.1.3 \
-    PyYAML==5.1.2 \
+	sagemaker-experiments==0.1.7 \
```

Comment:
Sorry for syntax error, will correct it before merging it.

Reply:
You'll have to fix any changes before the final approvals, else the approval is dismissed.